index.html

```diff
@@ -0,0 +1,72 @@
+	<link rel="stylesheet" type="text/css" href="./css/styles.css">
+</head>
+
+<body>
+	<nav class="mainnav">
+		<h1>Aspen Humphrey's Portfolio</h1>
+		<img class="mainphoto" src="./images/aspen.JPG">
+	</nav>
+
+	<section class="aspenbio">
+		<p>Hello! My name is Aspen Humphrey. I am a born and raised Nashvillian. I am currently on track to become a JR. Front End Web Developer through Nashville Software School's Cohort 20! I am highly motivated, intellegent, and looking for work with a team where my strong work ethic will be put to use! <br> I enjoy painting, reading, swimming, cooking, gardening, and making products with my harvests! I have 3 fur-babies and a loving supportive husband.</p>
+	</section>
+<br><br>
+
+	<!-- /**/<section class="resourceslinks">
+		<link rel="stylesheet" type="text/css" href="#">1
+		<link rel="stylesheet" type="text/css" href="#">2
+		<link rel="stylesheet" type="text/css" href="#">3
+	</section> -->
+<br><br>
+	<article class="project">
+		<section>future project placeholder1 </section>
+	<br>
+		<section>future project placeholder 2</section>
+	</article>
+<br><br>
+	<article id="blogposts">
+		<!-- <div class="posts">
+			<h2>Wonderland?</h2>
+			<p>Chasing what seems like an incredibly quick rabbit. Diving, Jumping, and Sprinting through tunnels- Could I be in Wonderland? <br> What is this place they call "NSS"? <br> Or is the better question to ask..What in the world is JavaScript? <br> When will I stop falling through the abyss, stop talking to the caterpillar who asks "Where are you", and finally catch the rabbit? <br> As I start feeling as if I am close to "catching" the rabbit (finally grasping the concept of javascript in its entirety) Tweedle Dee and Dum throw me for a loop once more and I find my head detached by the Red Queen. <br> By now I have officially turned into the Mad Hatter. As I accept my fate of eternally attending wild tea parties I also accept the phrase "let go and let the code happen".<p>
+		</div>
+	<br>
+		<div class="posts">
+			<h2>Zen: State of mind. </h2>
+			<p>Unity between body and mind. Or for me letting go (of control). But what does all of this mean? What does it take to really be "Zen"? This has been the toughest challenge I have faced thus far. Not only here at NSS (while navigating Wonderland)- but in my daily life. I thoroughly enjoy knowing what I am doing, when, how, and why- always. I am truly searching for my center in every aspect of my life; being in Cohort 20 has escalated this idea and has been changing my core with such force I have whiplash! I am becoming more and more uncomfortable with each passing day. As this level grows,  I have no choice but to let go of more and more. Thus being said- I am having a difficult time “letting go” and “trusting the method”- not just in NSS but in my life. I am very thankful for this change and for this opportunity. <br>
+
+		</div> -->
+	</article>
+
+<br><br>
+
+	<footer class="mainfooter">
+	
```

Comment:
Not sure we've talked about whitespace yet, but it can really cause some issues. It's best practice to get rid of it. You can change your settings in sublime to automatically trim the whitespace. Just enter this line into your custom settings.
`"trim_trailing_white_space_on_save": true,`
Also, a good way to visualize trailing whitespace is to add the trailing spaces package to your editor.